app/controllers/application_controller.rb

```diff
@@ -169,16 +169,25 @@ def error(message, now = false)
     flash_message(:error, message, now)
   end
 
-  def inline_error(message, now = false)
-    flash[:inline] = { :error => CGI.escapeHTML(message) }
+  def warning(message, now = false)
+    flash_message(:warning, message, now)
   end
 
-  def inline_success(message, now = false)
-    flash[:inline] = { :success => CGI.escapeHTML(message) }
+  def inline_error(message)
+    inline_flash_message(:error, message)
   end
 
-  def warning(message, now = false)
```

Comment:
should we deprecate this method instead? I'm uncertain if anyone is using it. also for consistency, what about other helper methods (error, notice etc).

Reply:
ApplicationShared#set_taxonomy uses it for sure, quick grep shows some plugins do too, see foreman-docker/app/controllers/containers_controller.rb